src/Components/CommentAdder.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from "react";
+import { postComment } from "../api";
+
+export const CommentAdder = ({ review_id, login, setComments }) => {
+  const [newComment, setNewComment] = useState("");
+  const [err, setErr] = useState(null);
+  const [isPosting, setIsPosting] = useState(false);
+
+  useEffect(() => {
+    setErr(null);
+  }, [login]);
+
+  const handleSubmit = (event) => {
+    setIsPosting(true);
+    setNewComment("");
```

Comment:
This feature doesnt appear to work, is it becasue newComment is set to blank here?